modules/ebi-metagenomics/jbrowse/sortgff/meta.yml

```diff
@@ -0,0 +1,56 @@
+name: sortgff
+description: sort gff file for jbrowse use
+keywords:
+  - gff
+  - jbrowse
+  - sort
+  - grep
+tools:
+  - grep:
+      description: Print lines matching a pattern
+      homepage: https://www.gnu.org/software/grep/
+      documentation: https://www.gnu.org/software/grep/manual/
+      licence: ["GPL-3.0-or-later"]
+      identifier: biotools:grep
+  - sort:
+      description: Sort lines of text files
+      homepage: https://www.gnu.org/software/coreutils/
+      documentation: https://www.gnu.org/software/coreutils/manual/html_node/sort-invocation.html
+      licence: ["GPL-3.0-or-later"]
+      identifier: biotools:sort
+input:
+  - - meta:
+        type: map
+        description: |
+          Groovy Map containing sample information
+          e.g. [ id:'test', single_end:false ]
+    - tab:
```

Comment:
I would use `gff` instead of tab

Reply:
I'll leave is as tab, as I'm using gff for the output. Changed the descrition to be more specific

Reply:
it should be gff, tab doesn't really convey what the input is... I would change the output to tbe sorted_gff or similar